File: Exercise434.java
/*(Reading numbers until a specified sum) Write an application that asks for a number from
the user and then keeps reading integer values from the user until the sum of those integers equals
or becomes greater than the value that was input in the beginning.
*/

import java.util.Scanner;

public class Exercise434{
	public static void main(String[] args){
		Scanner userInput = new Scanner(System.in);
			System.out.print("Enter a number: ")
			int number = userInput.nextInt();
			

			while (true){
				System.out.print("Enter a number or '*' to quit: ")
				int integer = userInput.nextInt().charAt();


			}
	}

}

File: Exercise4.34.js
const userInput = require("prompt-sync");
const prompt = userInput();

let number = Number(prompt("Enter a number: "));
let sum = 0;

while (true){
	let integer = Number(prompt("Enter a number or '0' to quit: "));

	if (integer == 0) break;
		sum += integer;
	if (sum >= number) break;
			}
	console.log("The sum is: " + sum);


